src/humanloop/decorators/prompt.py

```diff
@@ -0,0 +1,112 @@
+import uuid
+from functools import wraps
+from typing import Literal, Callable, Optional, Union
+
+from humanloop.otel import get_trace_context, get_tracer, pop_trace_context, push_trace_context
+from humanloop.otel.constants import HL_FILE_OT_KEY, HL_LOG_OT_KEY, HL_TRACE_METADATA_KEY
+from humanloop.otel.helpers import write_to_opentelemetry_span
+
+
+def prompt(
+    path: Optional[str] = None,
```

Comment:
Same point as before around having consistency in the args with the Prompt request if poss

Reply:
ack, fixed

Reply:
Realisation: a parameter is missing.

<img width="538" alt="image" src="https://github.com/user-attachments/assets/345d63d1-1f92-44c6-89b7-07e335ce42cf">

We should add it, since users might register a Prompt and Tools via decorators, then call `hl_client.prompts.call` expecting to receive tool calls.

With current setup, decorators create the File at first function call (Log creation). `linked_tools` expecting IDs, not paths. This inconsistency is surprising, as `linked_tools` should be able handle paths like other API points, just like e.g. `/prompts/log`

<img width="566" alt="image" src="https://github.com/user-attachments/assets/032bdd8a-4336-4b36-835a-02e6ead63424">

Reply:
there are also other fields missing like `attributes` and `tools`. Please do an audit to make sure there are no others missing and add the missing ones.

Please do this for all file types

Reply:
added missing attributes